lib/pf/cmd/pf/pfmon.pm

```diff
@@ -67,71 +67,8 @@ pf::cmd::pf::pfmon
 
 use strict;
 use warnings;
-use pf::config::pfmon qw(%ConfigPfmon);
```

Comment:
Why not just remove the file ?

Reply:
I left it for backward compatibility of cron jobs.